pixi.toml

```diff
@@ -0,0 +1,49 @@
+[project]
+name = "rerun"
+version = "0.9.0"
+description = "Add a short description here"
+authors = ["Jeremy Leibs <jeremy@rerun.io>"]
+channels = ["conda-forge"]
+platforms = ["linux-64", "osx-arm64", "win-64"]
+
+[tasks]
+
+codegen = "cargo codegen"
```

Comment:
Now we have three command systems: `just`, `cargo` and `pixi`. That's at least one too many imho. But I guess we can wait with porting `justfile` and `.cargo/config.toml` to a later PR

Reply:
You're forgetting `nox` 😅

Reply:
🤦